app/assets/javascripts/alchemy/alchemy.node_sorter.js

```diff
@@ -0,0 +1,60 @@
+Alchemy.DisplayNodeFolders = function() {
+  var generate_link = function(node_id, folded) {
+    var icon = folded === "true" ? 'plus' : 'minus';
+    return '<a class="node_folder" data-node-id="' + node_id + '"><i class="far fa-' + icon + '-square fa-fw"></i></a>'
+  }
+  document.querySelectorAll('li.menu-item').forEach(function (el) {
+    var leftIconArea = el.querySelector('.nodes_tree-left_images')
+    if (el.dataset.childrenCount > 0) {
+      leftIconArea.innerHTML = generate_link(el.dataset.nodeId, el.dataset.folded)
+    } else {
+      leftIconArea.innerHTML = '&nbsp;'
+    }
+  });
+};
+
+Alchemy.NodeSorter = function() {
+  Alchemy.DisplayNodeFolders();
+  document.querySelectorAll('#archive_all ul.children').forEach(function (el) {
```

Comment:
This will initialize this for a way to wide selector if you init this on every page load ;) (see note above)

Reply:
Fixed